modules/aws/elb_test.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+func getSubnetIdsPerAZ(t *testing.T, vpc *Vpc) []*string {
```

Comment:
@AlvaroRacero I think, it would make sense to add a comment that describes a reason behind this function. AFAIR, alb needs 2 subnets at least from a different availability zone